lib/signaling/v2/peerconnection.js

```diff
@@ -612,7 +611,7 @@ class PeerConnectionV2 extends StateMachine {
         // properties are immutable.
         description = {
           type: description.type,
-          sdp: isChrome && vp8SimulcastRequested
+          sdp: !isFirefox && vp8SimulcastRequested
```

Comment:
Should we explicitly enable vp8Simulcast for Chrome and Safari? `!isFirefox` may also enable it for other browsers like Edge in the future.